cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -103,8 +110,8 @@ spec:
       # osdsPerDevice: "1" # this value can be overridden at the node or device level
       # encryptedDevice: "true" # the default value for this option is "false"
 # Cluster level list of directories to use for filestore-based OSD storage. If uncommented, this example would create an OSD under the dataDirHostPath.
-    #directories:
-    #- path: /var/lib/rook
+# directories:
```

Comment:
Why did these lines change? The whitespace is intended to be that way so it's easy to uncomment and have it work

Reply:
reverted.